### PR TITLE
Use port 7080

### PR DESCRIPTION
**Pull request by jtart**
JIRA: https://jira.dev.bbc.co.uk/browse/NEWSART-562

Changes the running application to use port 7080 over port 3000. Updates docs.

- [x] JIRA ticket added
- [x] Pull request URL added to JIRA ticket
- [x] Up-to-date with latest - `git pull --rebase origin latest`
- [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
- [ ] ~~Tests added for new features~~
- [x] Tests pass - `npm test`

- [ ] Test engineer approval

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=7080
```
Comment (pjlee11) [inline .env:1]: Is the PORT variable needed on cosmos? Asking as it's not used elsewhere within this repo

Comment (jtart) [reply .env:1]: It's used for the PORT (`process.env.PORT`) here: https://github.com/bbc/simorgh/pull/20/files#diff-1fdf421c05c1140f6d71444ea2b27638R5. `razzle` sets the PORT to `3000` under the hood 👎

Comment (pjlee11) [reply .env:1]: Gotcha 👌

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Review by sareh — APPROVED** [review]
👍 Looks great & works for me locally.

**Pull request merged**